src/lib/firebaseBackend.js

```diff
@@ -472,15 +472,28 @@ class FirebaseBackend extends BackendInterface {
     window.localStorage.setItem('emailForSignIn', email);
   }
 
-  shouldRepromptEmail() {
-    return window.localStorage.getItem('emailForSignIn') === null;
+  getEmailForSignIn() {
+    return window.localStorage.getItem('emailForSignIn');
   }
 
-  async continueSignup(url, email, dropsite) {
+  async continueSignup(url, email) {
     if (this.firebase.auth().isSignInWithEmailLink(url)) {
-      var emailOrStoredEmail =
+      const emailOrStoredEmail =
         window.localStorage.getItem('emailForSignIn') || email;
```

Comment:
Do we want to check for a passed email first, using the stored email as a fallback?

Reply:
Actually yea, let's swap these.